lib/commands/doctor/index.js

```diff
@@ -18,7 +18,7 @@ class DoctorCommand extends Command {
             return Promise.reject(e);
         }
 
-        return this.ui.listr(checks, {ui: this.ui, system: this.system}).then(() => {
+        return this.ui.listr(checks, {argv: argv, ui: this.ui, system: this.system}).then(() => {
```

Comment:
I'd say we should remove this bit (and the related arguments) for now. Definitely 👍 with being able to skip it - but we can add it back in when we refactor the `ghost doctor` command (which will happen hopefully sometime this week)

Reply:
With this am I good to re-add the skip?

Reply:
Sorry for not responding to this until now - I'm good with re-adding the skip. However, that does mean that any command that _calls_ ghost doctor (e.g. `start`, `install`, `update`) will need to be updated to also pull in the required `ghost doctor` args.

Reply:
Done! 👍